### ISSUE TITLE
Opt-in email notifications when user gets a new 'message'

### ISSUE THREAD

**Issue by tog22** [tog22/eaforum#65]
Messages show at http://effective-altruism.com/message/inbox/ and include replies to a user's comments. Unfortunately people can't set this up themselves using http://effective-altruism.com/message/inbox/.rss because that only works when logged in. We're not prioritising this as its tricky.

(The old reddit codebase the forum's based on may contain this feature, or it may postdate the fork from reddit.)

**Comment by arichard4** [comment]
Old codebase lacks this feature, I think.

**Comment by patbl** [comment]
Sorry, I don't understand. Is the idea that we want to e-mail people when people reply to their comments? What does this have to do with that RSS feed?

**Comment by patbl** [comment]
Can someone help me understand this?

**Comment by tog22** [comment]
Your interpretation is right Patrick. You can ignore the reference to the RSS feed - I was simply saying that people can't subscribe to that RSS feed because it shows only when you're logged in.

(cc @georgiemallett, @peterhurford )

**Comment by peterhurford** [comment]
I think it would be good for people to get an email when they get a reply on the EA forum as this likely would increase engagement.

**Comment by peterhurford** [comment]
More than one person has actually been asking me about this, so I upped it to high priority.

**Comment by tog22** [comment]
Yes, that sounds sensible. Do you or @georgiemallett think it's valuable
and high priority enough to throw Harysh at it? It could be good to have
someone who's reliably able to contribute significant hours familiar with
the codebase. This is especially so in case there's a bug we need to fix
urgently to keep the forum running/usable, but I imagine that's unlikely.

**Comment by peterhurford** [comment]
Requested in http://effective-altruism.com/ea/169/a_response_to_ea_has_a_lying_problem/9rn

**Comment by georgiemallett** [comment]
re: Harysh - we should probably let CEA handle this since they're taking
over the forum right?

--
Georgie Mallett, Executive Director, .impact <http://dotimpact.im>

[image: Inline images 2] <http://dotimpact.im/>

**Comment by patbl** [comment]
You could ask Sam Deere about when they plan to start working on it. I think they're unlikely to push any new code to the forum in the next month, and at this point overlapping efforts or turf wars shouldn't be a concern. And a total replacement for the forum is a long way off, so this feature is probably worth implementing in any case.

**Comment by tog22** [comment]
I think @patbl is right, as we've discussed elsewhere I'd be surprised if CEA got a volunteer to do this any time soon, and quite likely wouldn't at all. So let's just keep making improvements! 🚀

**Comment by tog22** [comment]
@peterhurford / @georgiemallett  - it sounded like Patrick was OK to work on this, in which case you could mark him as the assignee?
